src/els_methods.erl

```diff
@@ -129,7 +129,17 @@ initialize(Params, State) ->
 
 -spec initialized(params(), state()) -> result().
 initialized(_Params, State) ->
-  {noresponse, State#{status => initialized}}.
+  %% Report to the user the server version
+  {ok,     App} = application:get_application(),
+  {ok, Version} = application:get_key(App, vsn),
+  lager:info("initialized: [App=~p] [Version=~p]", [App, Version]),
+  BinVersion = list_to_binary(Version),
+  Message = <<"Erlang LS version: ", BinVersion/binary>>,
+  Method  = <<"window/showMessage">>,
```

Comment:
There's also a `logMessage` method. Maybe that's less invasive than this one?

Reply:
I debated doing that, but figured we can use this for now and switch if it becomes a problem.

I like seeing what version I am running as a message, because when working on the server its something you want to be sure of while testing.